cfme/tests/intelligence/test_chargeback.py

```diff
@@ -19,89 +19,91 @@
     test_requirements.chargeback
 ]
 
-per_time = ['Hourly', 'Daily', 'Monthly', 'Weekly', 'Yearly'] if current_version() >= '5.7' \
-    else ['Hourly', 'Monthly']
-
-
-def new_compute_rate():
-    return cb.ComputeRate(description='cb' + fauxfactory.gen_alphanumeric(),
-                    fields={'Allocated CPU Count':
-                            {'per_time': random.choice(per_time), 'fixed_rate': '1000'},
-                            'Used Disk I/O':
-                            {'per_time': random.choice(per_time), 'fixed_rate': '10'},
-                            'Fixed Compute Cost 1':
-                            {'per_time': random.choice(per_time), 'fixed_rate': '100'},
-                            'Used Memory':
-                            {'per_time': random.choice(per_time), 'fixed_rate': '6000'},
-                            'Used CPU Cores': {'variable_rate': '0.05'}})
-
-
-def new_storage_rate():
-    return cb.StorageRate(description='cb' + fauxfactory.gen_alphanumeric(),
-                    fields={'Fixed Storage Cost 1':
-                            {'per_time': random.choice(per_time), 'fixed_rate': '100'},
-                            'Fixed Storage Cost 2':
-                            {'per_time': random.choice(per_time), 'fixed_rate': '300'},
-                            'Allocated Disk Storage':
-                            {'per_time': random.choice(per_time), 'fixed_rate': '6000'},
-                            'Used Disk Storage':
-                            {'per_time': random.choice(per_time), 'variable_rate': '0.1'}})
-
-
-def test_add_new_compute_chargeback():
-    ccb = new_compute_rate()
+
+def random_per_time():
```

Comment:
REQUIRED: support of CFME less than 5.7 has been dropped. Please remove unnecessary if